vm/src/obj/objset.rs

```diff
@@ -336,8 +346,10 @@ fn set_union(vm: &mut VirtualMachine, args: PyFuncArgs) -> PyResult {
     elements.extend(get_elements(other).clone());
 
     Ok(PyObject::new(
-        PyObjectPayload::Set {
-            elements: RefCell::new(elements),
+        PyObjectPayload::AnyRustValue {
```

Comment:
could we add an argument `elements` to `new_set()`? This probably could reduce all this duplication to
`Ok(vm.ctx.new_set(elements))`.